aiobotocore/paginate.py

```diff
@@ -59,12 +59,14 @@ def _init_pager(self):
         self._starting_truncation = 0
         self._inject_starting_params(self._current_kwargs)
 
-    @asyncio.coroutine
-    def next_page(self):
+    async def __aiter__(self):
+        return self
+
+    async def __anext__(self):
```

Comment:
code should be backward compatible, just do something like:
```
@asyncio.cororutine
def __anext__(self):
    ....
```
and replace `await` with `yield from`, also not need for now to remove`next_page` method.